.github/workflows/main.yml

```diff
@@ -62,3 +62,35 @@ jobs:
       # TODO: Enable verify-code-segments for Unix and Windows when it behaves the same as on macOS:
       # https://github.com/material-components/material-components-flutter-gallery/issues/565
       if: runner.os == 'macOS'
+
+  integration-test:
+    name: Integration test
+    runs-on: ubuntu-latest
+    steps:
+    # Set up Flutter and add it to the path.
+    - name: Clone Flutter repository with master channel.
+      uses: actions/checkout@v2
+      with:
+        repository: 'flutter/flutter'
+        ref: 'master'
+        path: 'flutter'
+        fetch-depth: 0
+    - name: Add Flutter to the PATH.
+      run: echo "::add-path::/$GITHUB_WORKSPACE/flutter/bin"
+    - name: Run Flutter doctor.
+      run: flutter doctor -v
+    - name: Enable Flutter web.
+      run: flutter config --enable-web
+
+    # Clone the Gallery repository under `code`, to avoid conflicts with `flutter`.
+    - name: Clone the Gallery repository.
+      uses: actions/checkout@v2
+      with:
+        path: 'code'
+
+    - name: Get packages for the Flutter project.
+      run: flutter pub get
+      working-directory: code
+    - name: Run Flutter integration tests.
```

Comment:
Could you rename this to 'Run gallery_compile_test' with a command for that test specifically? That way when we add more tests it'll be easier to view their outputs separately in the Github run result.

Reply:
That makes sense, will change it to that